_layouts/package_search.html

```diff
@@ -15,7 +15,7 @@
         <div class="input-group">
           <input id="search-query" autocomplete="off" type="text" class="form-control" name="q" placeholder="Search ROS packages">
           <span class="input-group-btn">
-            <button class="btn btn-default" type="submit"><span class="glyphicon glyphicon-search"></span></button>
+            <button class="btn btn-default" type="submit"><span class="glyphicon glyphicon-search" title="Find in rosindex"></span></button>
```

Comment:
This one is not showing for me, but it works if it's moved to `<button>`

Reply:
Oh, you're right. Just moved it, thanks!